interface-definitions/include/firewall/common-rule.xml.i

```diff
@@ -176,6 +176,9 @@
           <format>!&lt;MAC address&gt;</format>
           <description>Match everything except the specified MAC address</description>
         </valueHelp>
+        <constraint>
+          <validator name="mac-address-firewall"/>
```

Comment:
Why not use `<validator name="mac-address"/>`?

Reply:
We need to support ! prefix and `mac-address` is used in other modules.